src/style/smartListTheme.js

```diff
@@ -28,12 +30,18 @@ const smartListTheme = extendTheme({
       lineHeight: '110%',
       letterSpacing: '-2%',
       fontFamily: 'Montserrat',
-      background: 'brand.600',
+      bgImage: 'linear-gradient(to bottom,#76E4F7, #00A3C4)',
```

Comment:
I feel like I will get a headache with that gradient, but it is probably just me.